src/components/darkmode-switch/darkmode-switch.scss

```diff
@@ -0,0 +1,176 @@
+@import '../../style/functions.scss';
+
+:host {
+    --toggle-size: 1.25rem;
+    * {
+        box-sizing: border-box;
+    }
+}
+
+.mode-toggle {
+    margin: pxToRem(12);
+    position: relative;
+    width: var(--toggle-size);
+    height: var(--toggle-size);
+
+    transition: opacity 0.3s ease;
+    opacity: 0.5;
+
+    &:hover {
+        opacity: 1;
+    }
```

Comment:
I added this for the dark-mode button, because while the sun icon looks pretty at home next to the Kompendium logo, it popped _way_ too much next to the pale Lime Elements logo. Feel free to tweak @Kiarokh!

Reply:
Great job @adrianschmidt 
I think 50% opacity is a bit too brutal 😄 . But I'll check more.

Just found a Safari f-up 
![image](https://user-images.githubusercontent.com/35954987/124285319-191a0600-db4e-11eb-83de-b12b64b9c564.png)
Gonna fix this as well

Reply:
It's a bit too brutal next to the Kompendium logo, but I think it's pretty spot on when run in lime-elements to be honest…

If you install this locally in lime-elements, it works best with the `alt-kompendium-upgrade` branch in lime-elements.

Reply:
I like the opacity change. It provides more interactivity to the toggle as well. I just increased it to 70%. Hope it's OK for you.

Also made the heading a bit darker.

Rest of it looks great to me.

Reply:
Ah, yeah, much better! Very nice 👍

![image](https://user-images.githubusercontent.com/154725/124432804-4fd06600-dd72-11eb-9af7-ef001d3a65df.png)

(Old on the left, new on the right.)